CHANGELOG.md

```diff
@@ -8,6 +8,20 @@ and this repository adheres to [Semantic Versioning](https://semver.org/spec/v2.
 
 ## Unreleased
 
+## v1.2.0 - 2021-07-26
+
+### General
+
+- OpenTelemetry Java SDK and OpenTelemetry Instrumentation for Java dependencies were updated to version 1.4.0.
+
+### Enhancements
+
+- We have added instrumentations for [HikariCP](https://github.com/brettwooldridge/HikariCP)
+  and [Tomcat JDBC](https://tomcat.apache.org/tomcat-8.5-doc/jdbc-pool.html) connection pools. The agent will now
+  collect and export metrics for those two JDBC connection pools.
+- The agent now supports the `OTEL_SERVICE_NAME` environment variable (and `otel.service.name` system property) - you no
+  longer need to use `OTEL_RESOURCE_ATTRIBUTES` to set the service name.
```

Comment:
should we mention that this new way is a preferred one?

Reply:
Discussed in Slack.